app/ios/Runner/Assets.xcassets/LaunchImage.imageset/README.md

```diff
@@ -0,0 +1,5 @@
+# Launch Screen Assets
```

Comment:
- [ ] Make a note here to remove this file sometime, we'll keep for now but really not necessary